src/api/entities/SecurityToken/Compliance/Requirements.ts

```diff
@@ -187,7 +191,7 @@ export class Rules extends Namespace<SecurityToken> {
   private async _checkTransfer(args: {
```

Comment:
`from` should just be `string | Identity` now that there is no secondary issuance. Also, since `checkMint` will be deleted, this logic should be moved to `checkTransfer` and this function deleted as well